src/events/cd-event-list-item.vue

```diff
@@ -36,6 +32,14 @@
         {{ isFull ? $t('Full') : $t('See Details and Book') }}
       </router-link>
     </div>
+    <div v-if="isRecurring" class="cd-event-list-item__recurring-info">
+      <span class="fa fa-info-circle cd-event-list-item__recurring-info-icon"></span>
+      <span class="cd-event-list-item__recurring-info-header">{{ $t('This is a recurring event') }}</span>
+      <div class="cd-event-list-item__recurring-info-text">
+        {{ `${recurringFrequencyInfo} ${$t('at')}` }} {{ event.dates[0].startTime | cdTimeFormatter }} - {{ event.dates[0].endTime | cdTimeFormatter }},
```

Comment:
Oh, I didn't know that. Doesn't that mean next event in series is broken too since it finds the first date in the array that is after today but that may not necessarily be the soonest then?